modules/aggregation-layers/src/screen-grid-layer/screen-grid-layer.js

```diff
@@ -125,15 +126,9 @@ export default class ScreenGridLayer extends GridAggregationLayer {
 
     const {aggregationBuffer, maxBuffer, maxTexture} = this.state;
 
-    if (aggregationBuffer) {
-      aggregationBuffer.delete();
-    }
-    if (maxBuffer) {
-      maxBuffer.delete();
-    }
-    if (maxTexture) {
-      maxTexture.delete();
-    }
+    aggregationBuffer?.delete();
```

Comment:
Huh, I have only seen this used in function calls like `aggregationBuffer?.delete?.();`

I don't quite get how it works...

Does it equal to `(aggregationBuffer?.delete)();` (which could crash) or `aggregationBuffer?.(delete)());`

Reply:
`aggregationBuffer?.delete?.()` checks if `delete` exists before calling it.

In this particular context, if `aggregationBuffer` exists, it's always a Buffer object.

Reply:
I was worrying about the case where it doesn't exist - it wasn't clear to me that the `.?` construct in the middle would prevent evaluation of the function call operator at the end. I assumed it only applied to the field lookup. But I assume it does, that is great.